modules/Microsoft.Compute/diskEncryptionSets/.bicep/nested_roleAssignments.bicep

```diff
@@ -69,7 +69,7 @@ var builtInRoleNames = {
 }
 
 resource diskEncryptionSet 'Microsoft.Compute/diskEncryptionSets@2020-12-01' existing = {
-  name: last(split(resourceId, '/'))
+  name: last(split(resourceId, '/'))!
```

Comment:
What is this used for?

Reply:
Hey @ahmadabdalla this is the new character introduced by the PG to address this warning #2641 . The new Bicep version which is required for this change is rolled out to the GitHub agents now and hence I addressed the warning for my implementation here directly.

Reply:
Thanks for clarifying @AlexanderSehr . Learned something new today 😀

Reply:
No problem. It's a nasty nasty world out there :D